dev-python/statsmodels/statsmodels-0.10.2.ebuild

```diff
@@ -0,0 +1,74 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python2_7 python3_{5,6,7} )
```

Comment:
python3.5 has been removed. I'd also suggest dropping python2.7 but since it's an existing package, it's not so bad.